modules/nw-metallb-layer2-extern-traffic-pol.adoc

```diff
@@ -0,0 +1,21 @@
+[id="nw-metallb-layer2-extern-traffic-pol_{context}"]
+= Layer 2 and external traffic policy
+
+With layer 2 mode, one node in your cluster receives all the traffic for the service IP address.
+How your cluster handles the traffic after it enters the node is affected by the external traffic policy.
```

Comment:
ExternalTrafficPolicy - cluster vs local dictates how the cluster loadbalances to serve the incoming traffic to service.